iothub/device/src/Transport/Amqp/ClientWebSocketTransport.cs

```diff
@@ -358,7 +358,6 @@ private static void ValidateBufferBounds(byte[] buffer, int offset, int size)
             }
         }
 
-
```

Comment:
I also recommend updating the namespace this class is under from an Azure.Amqp namespace to one under Microsoft.Azure.Devices.Client.